README.md

```
@@ -1,124 +1,82 @@
 features.
 
 # Online Demo
 
 * A demo of the log viewer can be found at https://yscope.com/log-viewer
-* The demo loads a Hadoop YARN log file from the 
-  [hive-24hrs](https://zenodo.org/record/7094921#.Y5JbH33MKHs) log dataset. 
-  * More info on the dataset and other datasets can be found 
-    [here](https://github.com/y-scope/clp/blob/main/docs/Datasets.md).
-* To open an IR stream, drag and drop it onto the log viewer or use the open 
+* The demo loads a Hadoop YARN log file from the [hive-24hrs] log dataset.
+    * More info on the dataset and other datasets can be found [here][datasets].
```

Markdownlint

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)